ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -167,42 +169,60 @@ public class BottomSheetController: UIViewController {
         }
     }
 
-    /// Sets the `isHidden` property with a completion handler.
+    /// Changes the `isHidden` state with a completion handler.
     /// - Parameters:
     ///   - isHidden: The new value.
     ///   - animated: Indicates if the change should be animated. The default value is `true`.
     ///   - completion: Closure to be called when the state change completes.
     @objc public func setIsHidden(_ isHidden: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
-        guard isHidden != self.isHidden else {
-            return
-        }
-
+        let targetState: BottomSheetExpansionState = isHidden ? .hidden : .collapsed
         if isViewLoaded {
-            if isHidden {
-                panGestureRecognizer.isEnabled = false
-                move(to: .hidden, animated: animated, shouldNotifyDelegate: false) { [weak self] finalPosition in
-                    guard let strongSelf = self else {
-                        return
-                    }
-                    strongSelf.bottomSheetView.isHidden = true
-                    completion?(finalPosition == .end)
-                }
-            } else {
-                bottomSheetView.isHidden = false
-                move(to: .collapsed, animated: animated, shouldNotifyDelegate: false) { [weak self] finalPosition in
-                    guard let strongSelf = self else {
-                        return
-                    }
-                    strongSelf.panGestureRecognizer.isEnabled = strongSelf.isExpandable
-                    completion?(finalPosition == .end)
-                }
+            move(to: targetState, animated: animated) { finalPosition in
+                completion?(finalPosition == .end)
             }
         } else {
-            currentExpansionState = isHidden ? .hidden : .collapsed
+            currentExpansionState = targetState
             completion?(true)
         }
     }
 
+    /// Initiates an interactive `isHidden` state change driven by the returned `UIViewAnimating` object.
+    ///
+    /// The returned animator comes preloaded with all the animations required to reach the target `isHidden` state.
+    ///
+    /// You can modify the `fractionComplete` property of the animator to interactively drive the animation in the paused state.
+    /// You can also change the `isReversed` property to swap the start and target `isHidden` states.
+    ///
+    /// At the end of the interactive portion of the animation, you must call `startAnimation` on the animator to complete it non-interactively.
```

Comment:
What does "At the end of the interactive portion of the animation" mean for the calling code? Is there a way to be more precise on what needs to be done on their end? A small code snippet if possible?

Reply:
It means whenever they want to stop driving the animation (which they do by repeatedly setting fractionComplete whenever they want). I'll try to make it more clear.

Reply:
I reworded to make things more clear.